src/riak_test_escript.erl

```diff
@@ -429,3 +444,13 @@ so_kill_riak_maybe() ->
             io:format("Leaving Riak Up... "),
             rt:whats_up()
     end.
+
+is_common_test([]) -> 
+    false;
+is_common_test([{attribute, _, file, {File, _}} | T]) ->
+    case filename:basename(File) of
```

Comment:
No idea, I'm an erlang.mk guy ;)

rebar2 at least required the module name without the SUITE and then add it to the end to form the file name.